en/getting_started/README.md

```diff
@@ -15,11 +15,11 @@ It is [dual-licensed under Apache 2.0 and GPLv3](https://github.com/mavlink/qgro
 To get the source files:
 1. Clone the repo (or your fork) including submodules:
    ```
-   git clone https://github.com/mavlink/qgroundcontrol.git --recursive
+   git clone --recursive -j8 https://github.com/mavlink/qgroundcontrol.git
```

Comment:
I'd remove the `-j8`. Cloning is pretty fast anyway and I tend to think keeping it simple is better.

Reply:
> Cloning is pretty fast anyway and I tend to think keeping it simple is better.

Agree. People who are github pros can this however they want. These instructions are for newbies.